test-network-nano-bash/README.md

```diff
@@ -38,38 +38,78 @@ If you have [yq](https://mikefarah.gitbook.io/yq/) installed, run the following
 yq -i 'del(.chaincode.externalBuilders) | .chaincode.externalBuilders[0].name = "ccaas_builder" | .chaincode.externalBuilders[0].path = env(PWD) + "/builders/ccaas" | .chaincode.externalBuilders[0].propagateEnvironment[0] = "CHAINCODE_AS_A_SERVICE_BUILDER_CONFIG"' config/core.yaml
 ```
 
+## Specifying the type of consensus
+To specify the consesus type you need to alter the configuration file of the orderer and `configtx.yaml`.\
+The path specified to the configuration of the orderer is `fabric-samples/config/orderer.yaml`.
+You should edit the `Consensus` field in said file according to your needs.\
+For `BFT` consensus the configuration should look the following:
+
+```yaml
+Consensus:
+    WALDir: /tmp/e2e-smartbft-test3685389447/orderers/OrdererOrg.orderer1/etcdraft/wal
```

Comment:
can we put here strings similar to etcdraft? i.e not `/tmp/e2e-smartbft-test3685389447/` but something more similar to `/var/hyperledger/production/orderer/etcdraft/wal`